lib/api_hammer/faraday/request_logger.rb

```diff
@@ -79,8 +79,13 @@ def call(request_env)
               'activesupport_tagged_logging_tags' => @log_tags,
             }.reject{|k,v| v.nil? },
           }
-
-          json_data = JSON.generate(data)
+          begin
+            json_data = JSON.generate(data)
+          rescue Encoding::UndefinedConversionError => e
+            # force encoding on request data in case of a conversion error
+            data["request"].each {|k,v| v.force_encoding('UTF-8') if String == v.class }
```

Comment:
I don't much like just setting all strings to UTF8, that is not necessarily correct